tokio-console/src/state/mod.rs

```diff
@@ -107,6 +102,14 @@ impl State {
         current_view: &view::ViewState,
         update: proto::instrument::Update,
     ) {
+        match Temporality::try_from(update.temporality) {
```

Comment:
Perhaps best to stick to the way that we reference all other types from the `console_api` crate.
```suggestion
        match proto::instrument::Temporality::try_from(update.temporality) {
```